src/index.js

```diff
@@ -1,14 +1,43 @@
 import { createFilter } from '@rollup/pluginutils'
 
+var arraysEqual = function(a, b) {
```

Comment:
Do you mind moving helper functions to the bottom of the file?

Also, I prefer
```suggestion
function arraysEqual(a, b) {
```

Reply:
Would it be ok if I created a util.js (or similar) file?

Reply:
If not, I will move to the bottom of the file as suggested

Reply:
Bottom seems fine :-)